contracts/staking/contracts/src/libs/LibStakingRichErrors.sol

```diff
@@ -134,6 +134,10 @@ library LibStakingRichErrors {
     bytes4 internal constant INVALID_STAKE_STATUS_ERROR_SELECTOR =
         0xb7161acd;
 
+    // bytes4(keccak256("ProxyDestinationCannotBeNil()"))
+    bytes4 internal constant PROXY_DESTINATION_CANNOT_BE_NIL =
```

Comment:
I think we're trying to use this optimization for rich errors that don't have any parameters:
https://github.com/0xProject/0x-monorepo/blob/88e56356c424cc834948891ecb0918ced7c56bb9/contracts/exchange-libs/contracts/src/LibMathRichErrors.sol#L8